mrtt-ui/src/components/RestorationAimsForm/StakeholderBenefitsInputs.js

```diff
@@ -0,0 +1,86 @@
+import { useWatch } from 'react-hook-form'
+import PropTypes from 'prop-types'
+import React, { useEffect, useState } from 'react'
+
+import { FormLabel, MenuItem, Stack, TextField } from '@mui/material'
+import { makeValidCamelCaseName } from '../../library/strings/makeValidCamelCaseName'
+import { makeValidClassName } from '../../library/strings/makeValidClassName'
+import { NestedQuestion } from '../../styles/forms'
+import { restorationAims as questions } from '../../data/questions'
+import language from '../../language'
+const stakeholdersPropType = PropTypes.arrayOf(
+  PropTypes.shape({ stakeholderType: PropTypes.string, stakeholderName: PropTypes.string })
+)
+
+const StakeholderBenefitsInputs = ({
+  optionId,
+  parentQuestionFieldName,
+  reactHookFormInstance,
+  stakeholders
+}) => {
+  const questionLabelId = `${optionId}-question-label`
```

Comment:
we need the unique optionId generated based on options (from checkboxgroup) for accessibility. Each nested question needs a uniqueid.